app/src/main/java/com/intervaltimer/google/activities/WorkoutActivity.java

```diff
@@ -162,17 +170,49 @@ public void onClick(View view) {
         //getSupportActionBar().setDisplayHomeAsUpEnabled(true);
     }
 
+    private void resumeTimer() {
+        invertFabSelection();
```

Comment:
We are changing today's logic with this refactoring as we are calling invertFab... even if timerService is null... is this intentionally?